test/parallel/test-stream-readable-async-iterators.js

```diff
@@ -375,11 +375,26 @@ async function tests() {
     // eslint-disable-next-line no-unused-vars
     for await (const a of r) {
     }
-    // eslint-disable-next-line no-unused-vars
-    for await (const b of r) {
+    try {
+      // eslint-disable-next-line no-unused-vars
+      for await (const b of r) {
+      }
+    } catch (err) {
+      assert.strictEqual(err.code, 'ERR_STREAM_ITERATOR_EXISTS');
```

Comment:
Fwiw, this also doesn’t guarantee *that* an error was thrown here, because in that case the `catch() { ... }` block simply wouldn’t run. You typically want `assert.throws()` or, in this case, `assert.rejects()`, instead.